esmvalcore/cmor/_fixes/CMIP6/BCC_CSM2_MR.py

```diff
@@ -0,0 +1,66 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for BCC-CSM2-MR."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Common fixes to all vars"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes error in time coordinate, sometimes contains trailing zeros
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            try:
+                old_time = cube.coord('time')
+            except iris.exceptions.CoordinateNotFoundError:
+                return cubes
+
+            # if time variable is monotonic, there is nothing to do
+            if old_time.is_monotonic():
+                pass
+
+            time_units = old_time.units
+            time_data = old_time.points
+
+            idx_zeros = np.where(time_data == 0.0)[0]
+            time_diff = (time_units.num2date(time_data[1]) -
+                         time_units.num2date(time_data[0]))
+            days = time_diff.days
+
+            for idx in idx_zeros:
+                if idx == 0:
+                    continue
+                correct_time = time_units.num2date(time_data[idx - 1])
+                if days <= 31 and days >= 28:  # assume monthly time steps
+                    new_time = \
```

Comment:
Please try to avoid using backslashes, just make a long line and use `yapf -i` to get the correct formatting automatically.

Reply:
Done.